pkg/operator/target_config_reconciler.go

```diff
@@ -196,6 +202,16 @@ func (c TargetConfigReconciler) sync() error {
 		specAnnotations["serviceaccounts/openshift-descheduler-operand"] = resourceVersion
 	}
 
+	if stsa, _, err := c.manageSoftTainterServiceAccount(descheduler); err != nil {
+		return err
+	} else {
+		resourceVersion := "0"
+		if stsa != nil { // SyncConfigMap can return nil
```

Comment:
I think these comments regarding `SyncConfigMap` don't make sense in most of these.

Reply:
done